src/main/java/Serghei_Stoianov/Methods.java

```diff
@@ -0,0 +1,46 @@
+package Serghei_Stoianov;
+
+public class Methods {
+
+    //Write a method that takes two integers as parameters and returns the maximum of the two.
+    public static int MaxEl(int a, int b) {
+        if (a < b) a = b;
+        return a;
+    }
+
+    //  The method should take an integer as a parameter and return the factorial value.
+    static void Factorial(int num) {
+        int f = 1;
+        for (int i = 0; i++ < num; )
+            f *= i;
+        System.out.println("factorial value of entered number = " + f);
+    }
+
+    //Write a method that takes an array of integers as a parameter and returns the sum of all the elements in the array.
+    public static void SumOfArrayElements(int[] array) {
+        int sum = 0;
+        for (int i = 0; i < array.length; ) {
+            sum += array[i];
+            i++;
```

Comment:
please move it to its place in for loop

Reply:
thanks, now i've indersood that if i make i++< array.length -  i is incremented before solution, but if i increments i++ in the back of condition, i incremented after solution for.
resolved